### PR TITLE
Task3 done

### PR DESCRIPTION
**Pull request by nikmalevich**

### CODE REVIEW

File: UI/index.html [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+	</head>
+
+	<body>
+		<header>
+			<div class="logo">
+				<img src="logo.svg" class="logo">
+				<p class="logo">Tutter</p>
+			</div>
+			
+			<div class="logInOut">
```
Comment (MaksimViarenich) [inline UI/index.html]: For complex class name we use `-` as separator.
`logInOut` -> `log-in-out`

File: UI/styles.css [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+	position: absolute;
+	border-left: 5px solid #000000;
+	top: 130px;
+	bottom: 100px;
+	left: 15%;
+	height: calc(100% - 230px);
+}
+
+div.filters {
+	position: absolute;
```
Comment (MaksimViarenich) [inline UI/styles.css:135]: In general it's bad practice to use `absolute` position, try to work with `relative`

File: UI/styles.css [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+} 
+
+button.logInOut {
+	height: 80px;
+	width: 150px;
+	margin-top: -10px;
+	margin-right: 70px;
+	background-color: #E6F0F8;
+	border-radius: 50%;
+	border-width: 2;
```
Comment (MaksimViarenich) [inline UI/styles.css]: 2**px**

File: UI/index.html [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+				<p class="logInOut">nikmalevich <button class="logInOut">Log out</button></p>
+			</div>
+		</header>
+
+		<footer>
```
Comment (MaksimViarenich) [inline UI/index.html]: Better to move it to bottom. After <main></main>

File: UI/index.html [hunk 5]
```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
```
Comment (MaksimViarenich) [inline UI/index.html:1]: Try to change the height of the window, you will see this:
Снимок экрана 2020-03-02 в 23 33 21

Comment (MaksimViarenich) [reply UI/index.html:1]: And width also)
Снимок экрана 2020-03-02 в 23 34 56

File: UI/styles.css [hunk 6]
```diff
@@ -0,0 +1,264 @@
...
+	margin-top: 50px;
+	margin-left: 20%;
+	font-size: 40px;
+	font-weight: bold;
+}
+
+p.filterPosition {
+	margin-top: 10px;
+	margin-bottom: 0px;
+	margin-left: 10%;
```
Comment (MaksimViarenich) [inline UI/styles.css]: margin: 10px 0 0 10%;

**Comment by MaksimViarenich** [comment]
In general, it's good, just need to do some fixes to do it better.
You can ask any questions.

**Comment by nikmalevich** [comment]
Can I ask in lesson?

**Comment by MaksimViarenich** [comment]
Read, please, about `<form>`

**Comment by MaksimViarenich** [comment]
Add logic for delete/edit

**Review by MaksimViarenich — CHANGES_REQUESTED** [review]

File: UI/styles.css [hunk 7]
```diff
@@ -169,48 +190,62 @@ div.posts {
...
+	width: 80%;
 }
 
-p.postText {
-	margin-top: 0px;
-	margin-bottom: 0px;
-	margin-left: 3%;
-	font-size: 40px;
+p.post-tags, p.post-text, p.post-info {
+	margin: 5px 0 5px 0;
```
Comment (MaksimViarenich) [inline UI/styles.css:205]: margin: 5px 0;

File: UI/styles.css [hunk 8]
```diff
@@ -169,48 +190,62 @@ div.posts {
...
+button.like {
+	float: right;
+	margin-top: 70%;
+}
+
+button.delete {
+	display: inline-block;
+}
+
+button.edit {
```
Comment (MaksimViarenich) [inline UI/styles.css:241]: You can create one class for these buttons because they have the same style.

**Review by MaksimViarenich — APPROVED** [review]

**Pull request merged**